### PR TITLE
Added VAddy link in security section

### PR DESCRIPTION
**Pull request by ichikaway**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ If you're not inclined to make PRs you can tweet me at ```@ripienaar```
 
 ## Security and PKI
 
+  * http://vaddy.net - Continuous web security testing with continuous integration (CI) tools
```
Comment (ripienaar) [inline README.md]: do you mind adding some details, like '3 domains, 10 scan history for free' or similar

Comment (ichikaway) [inline README.md]: No problem. I added it.

**Comment by ripienaar** [comment]
sweet

**Comment by ichikaway** [comment]
Thank you :)

**Pull request merged**